### ISSUE TITLE
Add spritezero

### ISSUE THREAD

**Issue by mapsam** [mapbox/mapnik-swoop#7]
Let's add https://github.com/mapbox/spritezero to the swoop!

**Comment by mapsam** [comment]
`npm install spritezero --save` results in this error:

```
npm ERR! Error: ENOENT, chmod '/Users/mapsam/mapbox/mapnik-swoop/node_modules/spritezero/bin/spritezero'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Darwin 15.0.0
npm ERR! command "/Users/mapsam/.nvm/v0.10.40/bin/node" "/Users/mapsam/.nvm/v0.10.40/bin/npm" "install" "spritezero" "--save"
npm ERR! cwd /Users/mapsam/mapbox/mapnik-swoop
npm ERR! node -v v0.10.40
npm ERR! npm -v 1.4.28
npm ERR! path /Users/mapsam/mapbox/mapnik-swoop/node_modules/spritezero/bin/spritezero
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! not ok code 0
```

Trying to install it in a separate, test `package.json` results in a long, hanging install (has been trying for nearly 10 minutes).

/cc @jakepruitt @springmeyer

**Comment by jakepruitt** [comment]
Ahhh, I know what this is, I'll tackle it asap. Thanks for catching @mapsam.

**Comment by jakepruitt** [comment]
@mapsam should be fixed now with `spritezero@3.2.1`

**Comment by mapsam** [comment]
\o/

```
spritezero --> 
    ✓ spritezero tests  (1301ms)
```

Waiting on fixes at #8 before this goes in.

**Comment by springmeyer** [comment]
I've now got master passing. When trying to rebase`new-modules` against master I ran into nasty merge conflicts (I think due to the use of `npm --save` in the `new-modules` branch) so I committed directly: https://github.com/mapbox/mapnik-swoop/commit/83884a27994fe816f1011db721cb7ace069c6868
